### DISCUSSION TITLE
How to convert KMB to KML

### DISCUSSION THREAD

**Discussion by BernieNowel31** [organicmaps/organicmaps#5638] [Q&A]
Hello,
I hope someone can help me, I wanted to migrate to Organic maps but I lost all my bookmarks in maps.me
Fortunately I have all the bookmarks in KMB format but I can't do anything with them.
Does anyone know how to convert them or explain step by step how to do it?
Thank you in advance

**Comment by biodranik** [comment]
You can build and run kmb_to_kml converter in organicmaps repository, or send your kmb to me, I'll help. alex at organicmaps dot app

**Reply by BernieNowel31** [reply]
thank you for your reply.
how can i send you my files?

**Reply by biodranik** [reply]
I've downloaded your bookmarks, what is your email? It's better to write to my email in the above comment (alex at organicmaps dot app)

**Reply by BernieNowel31** [reply]
ok I send you an email

**Reply by biodranik** [reply]
Interestingly, our converter does not work. Looks like the format of kmb files was changed recently by mapsme. You may try to manually export each bookmark list and immediately open it with Organic Maps.

**Reply by CedricDViou** [reply]
Hello,
I'm trying to build and run kmb_to_kml to try to get my bookmarks back but I could only build the full organicmaps (kmb format doesn't show in import tab).
Can you guide me please?

**Reply by biodranik** [reply]
```
tools/unix/build_omim.sh -d kmb_to_kml
../omim-build-debug/kmb_to_kml <path_to_kmb_file>
```

**Reply by CedricDViou** [reply]
Thanks!
I get a Reader::SizeException: My file is probably not compatible.

**Reply by biodranik** [reply]
https://github.com/organicmaps/organicmaps/issues/5718

**Reply by lloritomars** [reply]
Can I also ask for a conversion from KMB to KML so I can use the Organic Maps. Thanks. I am sending to alex at organicmaps dot app.

**Comment by BernieNowel31** [comment]
ok bad luck for me ! Thanks for trying anyway.

**Reply by biodranik** [reply]
Why can't you manually export these files (using Share) from mapsme?

**Reply by BernieNowel31** [reply]
In fact, initially I thought I'd lost my bookmarks in maps.me because the application was empty and putting the KMBs here didn't change anything :
\smartphone\internal memory\MapsWithMe\bookmarks

But today I realize that the location of the application has changed :
\smartphone\internal memory\Android\data\com.mapswithme.maps.pro\files\MapsWithMe\bookmarks

I can now find my bookmarks in maps.me and export them one by one.
Another question : Do you know if organic maps will be designed to sort folders alphabetically or otherwise containing bookmarks?
I've waited so long for this feature in maps.me (which still doesn't work properly).

**Reply by biodranik** [reply]
Please follow #1066
